map/set-max-extent/README.metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "category": "Map",
+    "description": "Limit the view of a map to a particular area.",
+    "ignore": false,
+    "images": [
+        "SetMaxExtent.png"
+    ],
+    "keywords": [
+        "extent",
+        "limit panning",
+        "map",
+        "mapview",
+        "max extent",
+        "zoom",
+        "ArcGISMap",
+        "Envelope"
+    ],
+    "redirect_from": [
+        "/java/latest/sample-code/set-max-extent.htm"
+    ],
+    "relevant_apis": [
+        "ArcGISMap",
+        "Envelope"
+    ],
+    "snippets": [
+        "src/main/java/com.esri.samples.set_max_extent/SetMaxExtentSample.java"
```

Comment:
Originally this used forward slash but that caused the metadata checker to fail because it wanted the version with dots. Assume it's fine to ignore it in this case?

Reply:
that's interesting you saw a message for that: we'll want to update that in the script. This looks like this is passing, and matches the other json format.